src/hotspot/share/jfr/metadata/metadata.xml

```diff
@@ -582,13 +586,17 @@
     <Field type="ulong" contentType="bytes" name="inlinedBytes" label="Inlined Code Size" />
   </Event>
 
-  <Event name="CompilerPhase" category="Java Virtual Machine, Compiler" label="Compiler Phase" thread="true" >
+  <Event name="CompilerPhase" category="Java Virtual Machine, Compiler" label="Compiler Phase"
+         description="Describes various phases of the compilation process like inlining or string optimization related phases"
+         thread="true">
     <Field type="CompilerPhaseType" name="phase" label="Compile Phase" />
     <Field type="uint" name="compileId" label="Compilation Identifier" relation="CompileId" />
     <Field type="ushort" name="phaseLevel" label="Phase Level" />
   </Event>
 
-  <Event name="CompilationFailure" category="Java Virtual Machine, Compiler" label="Compilation Failure" thread="true"  startTime="false">
+  <Event name="CompilationFailure" category="Java Virtual Machine, Compiler" label="Compilation Failure"
+         description="In case a JIT compilation failed, a CompilationFailure is triggered, reporting the reason"
```

Comment:
It would be better to use "compilation failure" instead of the event name (CompilationFailure) here.

Reply:
Hi Erik, I adjusted the description.